src/BCPlayer.js

```diff
@@ -1,91 +1,149 @@
-import React, { Component } from 'react';
-import { StyleSheet, Platform } from 'react-native';
-import BrightcovePlayer from './BrightcovePlayer';
+import React, {Component} from 'react'
+import {Animated, BackHandler, Dimensions, Image, Platform, StatusBar, StyleSheet, Text} from 'react-native'
+import Orientation from 'react-native-orientation'
+import BrightcovePlayer from "./BrightcovePlayer";
+import PlayerEventTypes from "react-native-brightcove-player/src/PlayerEventTypes";
```

Comment:
@pankaj-pp any reason why not just use
`import PlayerEventTypes from "./PlayerEventTypes";`

Reply:
Yes , my bad. Will fix it